spec/services/types_generator_spec.rb

```diff
@@ -208,31 +255,6 @@
       end
     end
 
-    describe 'for subtypes' do
```

Comment:
this block of tests is no longer valid (we do not need to have a one-to-one mapping from work type to all of the subtypes as many will just come from the top level mapping, which is additive)